share/spice/tlds/tlds.js

```diff
@@ -0,0 +1,43 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_tlds = function(api_result) {
+        if(!api_result) {
+            return Spice.failed('namecheap');
+        }
+        var item = api_result.ApiResponse.CommandResponse.gettldlist;
+        if(!item) {
+            return Spice.failed('namecheap');
+        }
+        /* extract data from JSON */
+        
+        var tldName = item.tldName;
+        var tldType = item.tldType;
+       
+        var data = {
```

Comment:
Looks like this still needs to be simplified: https://github.com/duckduckgo/zeroclickinfo-spice/pull/1820#discussion_r31871580